lib/helpers.dart

```diff
@@ -0,0 +1,22 @@
+import 'dart:async';
+import 'package:connectivity/connectivity.dart';
+import 'package:data_connection_checker/data_connection_checker.dart';
+
+Future<List> getGPS(gpsObject) async {
+    var newLocation = await gpsObject.getlocation();
+    return [newLocation.latitude, newLocation.longitude];
+  }
+
+
+  Future<bool> isConnected() async {
+    var connectivityResult = await (Connectivity().checkConnectivity());
+    if (connectivityResult == ConnectivityResult.mobile) {
+      return (await DataConnectionChecker().hasConnection);
+    } 
+    else if (connectivityResult == ConnectivityResult.wifi) {
+      return (await DataConnectionChecker().hasConnection);
+    } 
+    else {
+      return false;
+    }
+  }
```

Comment:
# Try using this instead 😁
your code is a little janky. i wont way it wont work but reducing the amount of lines to something a little more intuitive would work better
```dart
  Future<bool> isConnected() async {
    var connectivtyResult = await Connectivity().checkConnectivity();
    print(connectivtyResult != ConnectivityResult.none ? "CONNECTED" : "NOT CONNECTED");
    return connectivtyResult != ConnectivityResult.none ? true : false;
  }
```